backend/src/Command.ts

```diff
@@ -90,12 +91,13 @@ export default class Command {
         result = stdout.toString()
       }
     } catch (error) {
-      if (this.isErrorReportable(error)) {
+      if (this.isErrorReportable(error) && !cli.data.errorsCode.includes(error.code)) {
```

Comment:
What would you think of putting this check inside the `isErrorReportable` function?